File: 016_dates/dates.py
#!/usr/bin/env python

from datetime import date
import sys

for i, line in enumerate(sys.stdin):

    year, month = [int(num) for num in line.rstrip().split('-')]
    month += 1

    current = date(year, month, 28)
    print current.strftime("%^A") 

File: 016_dates/dates.rb
#!/usr/bin/env ruby

require 'date'

ARGF.each_with_index do |line, idx|

    year, month = line.split('-')
    current = Date.new(year.to_i, month.to_i + 1, 28) 
    puts current.strftime("%^A")
end
